server/tests/audit_math/test_macro.py

```diff
@@ -136,6 +136,76 @@ def test_max_error(contests, batches):
             )
 
 
+def test_get_sample_sizes(contests, batches):
+    expected_first_round = {
+        "Contest A": 29,
+        "Contest B": 15,
+        "Contest C": 10,
+    }
+
+    sample = {}
+    for contest in contests:
+        computed = macro.get_sample_sizes(
+            RISK_LIMIT, contests[contest], batches, sample
+        )
+
+        assert (
+            expected_first_round[contest] == computed
+        ), "First round sample expected {}, got {}".format(
+            expected_first_round[contest], computed
+        )
```

Comment:
in my experience, the assertion error messages are pretty clear without writing a custom error message (they have really good diffs and line numbers attached)

Reply:
if it's useful to you, no prob